pkg/pipelines/deployment/deployment.go

```diff
@@ -91,10 +95,12 @@ func ptr32(i int32) *int32 {
 	return &i
 }
 
-func labelSelector(name, value string) *metav1.LabelSelector {
+//LabelSelector used to create the labelSelector for the commit status tracker
```

Comment:
This isn't specific for "commit status tracker"   I would change the comment "for a deployment".